components/admin_console/admin_definition.jsx

```diff
@@ -3164,6 +3164,15 @@ export default {
                         help_text_default: 'When true, webhooks, slash commands and other integrations, such as [Zapier](!https://docs.mattermost.com/integrations/zapier.html), will be allowed to change the profile picture they post with. Note: Combined with allowing integrations to override usernames, users may be able to perform phishing attacks by attempting to impersonate other users.',
                         help_text_markdown: true,
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'ServiceSettings.CreateBotAccounts',
+                        label: t('admin.service.createBotTitle'),
+                        label_default: 'Enable Bot Accounts',
+                        help_text: t('admin.service.enableBotAccounts'),
+                        help_text_default: 'When true, bot accounts can be used for integrations in Integrations > Bot Accounts. Bot accounts are similar to user accounts except they cannot be used to log in. See documentation to learn more.',
+                        help_text_markdown: true,
```

Comment:
There's no actual markdown in the help text, so I think this can be `false`.

Reply:
It was intended to use Markdown to include a link in the help text. I've clarified this in my comment above https://github.com/mattermost/mattermost-webapp/pull/2751#discussion_r282890875

So we can keep this value as `true`.